user-org-keycloak-util/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <groupId>org.sunbird</groupId>
+   <artifactId>user-org-keycloak</artifactId>
+   <version>1.0-SNAPSHOT</version>
+   <modelVersion>4.0.0</modelVersion>
+   <url>http://maven.apache.org</url>
+   <name>user-org-keycloak</name>
+   <properties>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
+      <log4j.version>2.8.2</log4j.version>
+      <junit.version>4.12</junit.version>
+      <powermock.version>1.6.5</powermock.version>
+   </properties>
+   <dependencies>
+      <dependency>
+         <groupId>org.sunbird</groupId>
+         <artifactId>user-org-util</artifactId>
+         <version>1.0-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+         <groupId>org.keycloak</groupId>
+         <artifactId>keycloak-admin-client</artifactId>
+         <version>6.0.1</version>
+      </dependency>
+      <dependency>
+         <groupId>org.jboss.resteasy</groupId>
+         <artifactId>jaxrs-api</artifactId>
+         <version>3.0.11.Final</version>
+      </dependency>
+      <dependency>
+         <groupId>org.jboss.resteasy</groupId>
+         <artifactId>resteasy-client</artifactId>
+         <version>3.1.0.Final</version>
+      </dependency>
+      <dependency>
+         <groupId>org.jboss.resteasy</groupId>
+         <artifactId>resteasy-jackson2-provider</artifactId>
+         <version>3.1.3.Final</version>
+         <exclusions>
+            <exclusion>
+               <groupId>com.fasterxml.jackson.core</groupId>
+               <artifactId>jackson-annotations</artifactId>
+            </exclusion>
+            <exclusion>
+               <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Already defined in user-org-util pom.xml

Reply:
com.fasterxml.jackson.core is  exclusion from  org.jboss.resteasy and this is not define in user-org-util